lerna.json

```diff
@@ -3,7 +3,8 @@
   "command": {
     "publish": {
       "ignoreChanges": [
-        "packages/package-json-lint/**"
+        "packages/package-json-lint/**",
+        "packages/react-config-terra/**"
```

Comment:
What's this?

Reply:
I don't want react-config-terra to release until we do a release, not just on first check in.

Reply:
so we should remove this code whenever we do the release
this and the 'private' in the package.json